src/hb-ot-tag.cc

```diff
@@ -45,6 +45,8 @@ hb_ot_old_tag_from_script (hb_script_t script)
     /* KATAKANA and HIRAGANA both map to 'kana' */
     case HB_SCRIPT_HIRAGANA:		return HB_TAG('k','a','n','a');
 
+    case HB_SCRIPT_MATH:		return HB_TAG('m','a','t','h');
```

Comment:
`hb_ot_old_tag_to_script` uses `HB_OT_MATH_SCRIPT` so this should too.

Reply:
Thanks, corrected.